src/main/java/frc/robot/subsystems/SimulationSubsystem.java

```diff
@@ -46,8 +61,15 @@ public void simulationPeriodic() {
     //Update simYaw distance
     simYaw += chassisRotationSpeed *  Robot.kDefaultPeriod;
 
+    //Update simPitch
+    distance = swerveSubsystem.getVelocity() / 60;
```

Comment:
Use swerveSubsystem.getPose().getX() for distance.

Reply:
Completed